Preparation/MyStack/MyStackInter.cs

```diff
@@ -1,34 +1,27 @@
 using System;
 using System.Collections;
+using System.Data;
 
 namespace MyStack
 {
-    public class MyStack
+    public class MyStack : IEnumerable
     {
-        public static readonly int maxsize = 10;
+        public static int count = 1;
         public int topstack;
-        public int[] arrayint = new int[maxsize];
+        public int[] arrayint = new int[count];
```

Comment:
Should be private, not public. As well as other fields

Reply:
Btw, it is a bad idea naming fields depending on their types. I would name it `underlyingArray`